Python/Product/VSCode/AnalysisVsc/LanguageServer.cs

```diff
@@ -137,14 +137,9 @@ private void OnUnregisterCapability(object sender, UnregisterCapabilityEventArgs
             => _rpc.NotifyWithParameterObjectAsync("client/unregisterCapability", e.@params).DoNotWait();
         #endregion
 
-        #region Cancellation
-        [JsonRpcMethod("cancelRequest")]
```

Comment:
This method still exists - the IDE will send it to cancel whatever request is currently running. How do we cancel any of the requests without it?

Reply:
Oh I see, this gets handled specially by StreamJsonRpc now. Fair enough.